rust/hyperlane-core/src/accumulator/merkle.rs

```diff
@@ -15,12 +14,42 @@ use crate::{
 //    - remove ring dependency
 // In accordance with its license terms, the apache2 license is reproduced below
 
-lazy_static! {
-    /// Zero nodes to act as "synthetic" left and right subtrees of other zero nodes.
-    pub static ref ZERO_NODES: Vec<MerkleTree> = {
-        (0..=TREE_DEPTH).map(MerkleTree::Zero).collect()
-    };
-}
+/// Zero nodes to act as "synthetic" left and right subtrees of other zero nodes.
+pub const ZERO_NODES: [MerkleTree; TREE_DEPTH + 1] = [
```

Comment:
I'm no macro expert but if you happen to be then this may be a good opportunity for one. If it's a hassle no worries

A comment indicating why we decided to write this out manually instead of `lazy_static` would also be useful for future readers